### ISSUE TITLE
Webmin don't show all active IPv4 addresses, keeps going offline. I cant even recovery with OVH rescure

### ISSUE THREAD

**Issue by TheBudda** [webmin/webmin#1506]
OS ubuntu: 20.04.2
webmin: 1.979
Authentic theme version: 19.81

My dedicated server is with OVH, I have been with OVH for 12 years. I have never had this issue in the past with adding IPs with OVH or webmin. When adding blocks of IPs. This server can have a max of 128 IPs, she is a beast!!

The Webmin GUI doesn't display all addresses
![network_interfaces](https://user-images.githubusercontent.com/16983376/128635812-8668e15e-3148-4814-bc5f-fd6014044f42.png)

I have had to reinstall ubuntu 20.04.2, 5x now, I am not a happy bunny. The network drops out after a bit of time after adding blocks of IPs via webmin >> Networking >> Network Configuration >> Network Interfaces >> Add a new interfaces
![network_configuration_network_interfaces](https://user-images.githubusercontent.com/16983376/128636476-3e240a8e-2bbb-4919-84d3-0a771b9f1fc8.png)

/etc/network/interfaces.d  folder is empty, due to using netplan
/etc/network/interfaces is attached in .txt and showing nothing of importance 
[interfaces.txt](https://github.com/webmin/webmin/files/6950980/interfaces.txt)

Here is part of my 50-cloud-init.yaml in .txt You can see the added IPs by webmin. I can supply the whole 50-cloud-init.yaml file if needed
[50-cloud-init.yaml.txt](https://github.com/webmin/webmin/files/6950974/50-cloud-init.yaml.txt)
               
Why on the webmin home page all IP's are not showing and the IPs that do show are in the wrong order? 
But, all IPs show under webmin >> Networking >> network configuration >> network interfaces are showing? 
Why does the server keep going offline?
It is driving me crazy!! Please see screenshots! Please help

Kind regards
TheBudda

**Comment by jcameron** [comment]
Do the IPs show up on the "Active Now" tab?

If not, what does the `ip addr` command output on your system?

**Comment by jcameron** [comment]
That does look like a bug, as I do see the IPs in the output from `ip addr`, and that's the command Webmin reads.

I see the cause though, and will include a fix in the next Webmin release.

**Comment by TheBudda** [comment]
Hi, @jcameron I have just updated webmin to 1.980 and this issue is still here. I can only see 5 IPs in Dashboard, Network Interfaces like the above screenshot and the screenshot below
![webmin_network_interfaces](https://user-images.githubusercontent.com/16983376/130543054-a1b2b92e-b80c-4bf2-ae47-84a47ebdd0e0.png)

command ip addr is still the same
![1p addr](https://user-images.githubusercontent.com/16983376/130543219-6b4c352d-d95b-4e6b-a4c8-94b2b4e5f47b.png)

ssh into server all IPs are shown, still the same
![ssh_output](https://user-images.githubusercontent.com/16983376/130543495-f8291862-85dd-4b1f-a530-d2520cbcc86c.png)
IP addr still the same
![1p addr](https://user-images.githubusercontent.com/16983376/130543219-6b4c352d-d95b-4e6b-a4c8-94b2b4e5f47b.png)

I have changed nothing on the server since I opened this issue. I have rebooted the server and still no joy!! If you look at the output of the command, ip addr then look at, Dashboard, Network Interfaces, all IP's are not even in the right order. And, I am still missing 5 IPs in Dashboard, Network Interfaces

Regards
TheBudda
